src/server/ApiRoutes/Bib.js

```diff
@@ -230,4 +231,5 @@ export default {
   fetchBib,
   nyplApiClientCall,
   addLocationUrls,
+  appendDimensionsToExtent,
```

Comment:
Seems weird that we're exporting `appendDimensionsToExtent` ?

Reply:
its for testing

Reply:
But it's not defined in that file anymore. May have been a bad merge. Doesn't matter much